### PR TITLE
Update installation instructions for v5

### LINKED ISSUE #168: Can you add Mac OSX install instructions for pester?
Can you add Mac OSX install instructions for pester?

Here is the basic outline of how to do it:

1.  brew install powershell
2.  pwsh
3.  Find-Module pester -Repository psgallery | Install-Module -MinimumVersion 4.3.1 -Force -SkipPublisherCheck -Scope CurrentUser
4.   Import-Module Pester -PassThru
5.   Invoke-Pester .\infrastructure_tests\buildValidation.Tests.ps1

### LINKED ISSUE #298: Installation page seems dated: doesn't cover beyond win2k16, win10, ps5
### Checklist

- [X] Issue has a meaningful title
- [X] I have searched the existing issues. See [issues](https://github.com/pester/docs/issues)

### Link to page

https://pester.dev/docs/introduction/installation

### What is the issue?

As elaboration on the issue title, consider this paragraph (and section titles that follow):
Compatibility
_Pester is compatible with every version of Windows that can run at least PowerShell 3. That includes: Windows 10, 8, 7, Vista, and their respective Server versions 2016, 2012 R2, 2012, 2008 R2 and 2008._

### Suggested fix?

Update page (this paragraph and sections that follow) for later versions, confirming those which are indeed supported (or if old versions of Windows or PS might no longer  be supported).

### PR DESCRIPTION
**Pull request by fflaten**
Removes outdated mentions of Windows OS-versions, simplifies installation instructions and include Linux / MacOS instructions.

Fix #168
Fix #298

### CODE REVIEW

File: docs/introduction/installation.mdx [hunk 1]
```diff
@@ -7,130 +7,115 @@ description: Pester is a cross-platform PowerShell module for testing your Power
...
 
+Run the command below to install the latest version:
 ```powershell
-$module = "C:\Program Files\WindowsPowerShell\Modules\Pester"
-takeown /F $module /A /R
-icacls $module /reset
-icacls $module /grant "*S-1-5-32-544:F" /inheritance:d /T
-Remove-Item -Path $module -Recurse -Force -Confirm:$false
+# -Force to install side-by-side
+# -SkipPublisherCheck to accept the newer certificate
```
Comment (fflaten) [inline docs/introduction/installation.mdx:45]: Publisher check error was PowerShellGet 1.6.5 shipped in PowerShell 6.1 and later.
Still including the `-SkipPublisherCheck` switch as all versions of Windows ship with an older PowerShellGet 1.0.0.1 version for Windows PowerShell.

Comment (fflaten) [inline docs/introduction/installation.mdx:45]: Not required using PowerShellGet 1.6.5 which shipped with PowerShell 6.1.

Still including this as all versions of Windows ship with an older PowerShellGet 1.0.0.1 version for Windows PowerShell.

File: docs/introduction/installation.mdx [hunk 2]
```diff
@@ -7,130 +7,115 @@ description: Pester is a cross-platform PowerShell module for testing your Power
...
-Install-Module -Name Pester -Force -SkipPublisherCheck
-```
-
-The `-SkipPublisherCheck` option is typically required for machines that only have v3.4.0 installed as mentioned above.
-That particular version of the module was signed by Microsoft to be shipped in-box, which is not the case for later versions which are community maintained and signed with a different certificate.
-Attempting to install the module without that option may fail.
-
-If you receive the following warning when trying to install the module, you may need to explicitly enable TLS 1.2:
+# For all users
+Install-Module -Name Pester
```
Comment (nohwnd) [inline docs/introduction/installation.mdx:22]: CurentUser scope is (largely) the default nowadays. Maybe change this to say # Install to default location ? and then optionally show how to install for all users or current user, or maybe just link to official doc, we don't need to repeat that here.

from [doc](https://learn.microsoft.com/en-us/powershell/module/powershellget/install-module?view=powershellget-3.x): When no Scope is defined, the default is set based on the PowerShellGet version.

In PowerShellGet 1.x versions, the default is AllUsers, which requires elevation for install.
For PowerShellGet versions 2.0.0 and above in PowerShell 6 or higher:
The default is CurrentUser, which doesn't require elevation for install.
If you are running in an elevated session, the default is AllUsers.

File: docs/introduction/installation.mdx (hunk 1 above)
Comment (nohwnd) [reply docs/introduction/installation.mdx:45]: 👍

File: docs/introduction/installation.mdx [hunk 3]
```diff
@@ -7,130 +7,115 @@ description: Pester is a cross-platform PowerShell module for testing your Power
...
-
-```powershell
-Update-Module -Name Pester
-```
+Alternatively, you can install the module manually using the instructions below.
 
 ## Installing manually
 
-To install Pester you don't have to use any package manager. Downloading and installating it manually can be useful in e.g. offline envionments.
+You don't have to use a package manager to install Pester. Another option is to download and installating it manually which can be useful in e.g. offline envionments.
```
Comment (nohwnd) [inline docs/introduction/installation.mdx:95]:
```suggestion
You don't have to use a package manager to install Pester. Another option is to download and installing it manually which can be useful in e.g. offline environments.
```

File: docs/introduction/installation.mdx (hunk 2 above)
Comment (fflaten) [reply docs/introduction/installation.mdx:23]: Thanks for the quick review! 🙂
Better in the latest commit?

**Comment by bravo-kernel** [comment]
@nohwnd please merge when :)

**Pull request merged**